src/einsteinpy/metric/kerr.py

```diff
@@ -18,7 +18,20 @@ class Kerr:
     """
 
     @u.quantity_input(time=u.s, M=u.kg)
-    def __init__(self, bl_coords, M, time):
+    def __init__(self, bl_coords, M, time=0 * u.s):
+        """
```

Comment:
`__init__` method is not meant to be used normally, as we have made 2 other constructor methods for initialization

Reply:
okay I have uninitialised time from 0.
Is it okay now?

Reply:
What I meant is `__init__` don't need docstrings for all three classes under `metric`

Reply:
But they are there in classes like velocity.py and core.py

Have removed them though as you said

Reply:
They don't have explicit classmethods.
Thanks @rtanmay for the changes.
I would review it extensively in a while